pkg/grub/rootfs.cfg

```diff
@@ -102,19 +102,24 @@ function set_to_existing_file {
 }
 
 function set_grub_virt {
-   smbios -t 1 -s 0 --set smb_vendor
-   if [ "$smb_vendor" == QEMU ]; then
-     set_global grub_virt qemu
-   elif [ "$smb_vendor" == Google ]; then
+   set arch=${grub_cpu}
```

Comment:
There's `set arch=${grub_cpu}` on line 291 before this functions gets called -- you don't need it here.

Reply:
Okay, will remove it.